libraries/core-react/stories/components/Tooltip.stories.tsx

```diff
@@ -149,14 +185,19 @@ export const TableCellsWithTooltip: Story<TooltipProps> = () => {
     openCell: null,
   })
 
+  let timer: ReturnType<typeof setTimeout>
```

Comment:
Should have default value, maybe just set it to null?

Reply:
Those icons are actually Equinor icons 😄